website/src/components/flex/FlexHero.js

```diff
@@ -1,7 +1,23 @@
+import Image from "next/image";
 import styles from "./FlexHero.module.scss";
 
-const FlexHero = () => {
-  return <div className={styles.outer}>Hero</div>;
+const FlexHero = (props) => {
+  return (
+    <div className={styles.outer}>
+      <h3>Hero</h3>
+
+      {props.heroImage?.sourceUrl && (
+        <div className={styles["hero-image"]}>
+          <Image
+            src={props.heroImage.sourceUrl}
+            width={props.heroImage.mediaDetails.width}
```

Comment:
This assume we want to use the hero image dimensions, but what if we want to force a crop to a size (like width)? I could see us wanting to do both.

Reply:
Next.js' Image component does not support cropping to different sizes at this time - there's support for this layout mode option called "fill" which is documented here: https://image-component.nextjs.gallery/layout-fill. I don't think that solves the hero image problem. For now, if cropping is important, we'll just need to build our own cloudinary URL as we did before, or let the image exist in its intrinsic dimensions inside of a div with fixed width/height and overflow hidden and use `transform:translate(-50%,-50%)` to center the image in that box

Reply:
let's leave it for now, but as we get going we're definitely going to support these scenarios:

1) We want to only specify width in the template. The component should send cloudinary a width param, then figure out a height based on known ratio we get from graphql.

1b) Same as above, but for height.

2) We want to force everything into a certain ratio. Think pictures on a team page, where each should be 1:2 ratio.

2b) If cloudinary, we should be able to pass in a param to center on faces